app/assets/javascripts/models/page/search_page.js.coffee

```diff
@@ -50,7 +50,7 @@ ns.SearchPage = do (ko
     $('.launch-customize-modal').click ->
       $('#customizeDataModal').modal('show')
 
-    setTimeout initModal, 2000 if !window.edscportal
+    preferences.ready.done(-> initModal()) if !window.edscportal
 
   class SearchPage
     constructor: ->
```

Comment:
>>"The "evidence" of multiple initializations can be seen in the js console. Everytime the preferences is initialized, an ajax call is made which is logged in the console."

I was asking for evidence that the site tour was initialized more than once.  I acknowledge that preferences is instantiated multiple times; however, I've explained why that is by design.  It is also not breaking anything.